wp/headless-wp/includes/classes/Feed.php

```diff
@@ -21,7 +21,7 @@ public function register() {
 		add_filter( 'the_content_feed', [ $this, 'filter_url' ], 999 );
 		add_filter( 'the_excerpt_rss', [ $this, 'filter_url' ], 999 );
 		add_filter( 'comments_link_feed', [ $this, 'filter_url' ], 999 );
-		add_filter( 'commnet_link', [ $this, 'filter_url' ], 999 );
```

Comment:
This one is a bit more significant as the hooked function wouldn't have worked before.

Reply:
Good Catch!